.travis.yml

```diff
@@ -7,7 +7,11 @@ matrix:
     - node_js: 10
     - node_js: 12
 
+before_install:
+  - npm install -g npm@7
+
 script:
+- npm --version
```

Comment:
Did you add this for testing or did you intend to leave it in there? I'm fully good with either, just thought I'd check

Reply:
I thought it would be good to leave there so we can see for sure which version of npm is being used prior to running the rest of the "build" commands.